lib/katello/version.rb

```diff
@@ -1,3 +1,3 @@
 module Katello
-  VERSION = "4.15.0-master".freeze
+  VERSION = "4.15.0.rc1".freeze
```

Comment:
Should this be rc2?

Reply:
Yeah, probably. I might have not set it properly at some point